packages/cell-toolbar/src/celltoolbartracker.ts

```diff
@@ -0,0 +1,372 @@
+/* -----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+import { Cell, ICellModel, MarkdownCell } from '@jupyterlab/cells';
+import { DocumentRegistry } from '@jupyterlab/docregistry';
+import { Notebook, NotebookPanel } from '@jupyterlab/notebook';
+import {
+  IObservableList,
+  IObservableUndoableList
+} from '@jupyterlab/observables';
+import { ISettingRegistry } from '@jupyterlab/settingregistry';
+import { LabIcon } from '@jupyterlab/ui-components';
+import { each } from '@lumino/algorithm';
+import { CommandRegistry } from '@lumino/commands';
+import { IDisposable } from '@lumino/disposable';
+import { PanelLayout, Widget } from '@lumino/widgets';
+import { CellToolbarWidget } from './celltoolbarwidget';
+import { EXTENSION_ID, ICellMenuItem } from './tokens';
+
+// icon svg import statements
+import addAboveSvg from '../style/icons/addabove.svg';
+import addBelowSvg from '../style/icons/addbelow.svg';
+import deleteSvg from '../style/icons/delete.svg';
+import duplicateSvg from '../style/icons/duplicate.svg';
+import moveDownSvg from '../style/icons/movedown.svg';
+import moveUpSvg from '../style/icons/moveup.svg';
+
+const DEFAULT_LEFT_MENU: ICellMenuItem[] = [];
```

Comment:
It would make sense to set the default list matches the one in the settings (in case of simpler remix not using the settings plugin).